keps/sig-network/0031-20181109-multicast-support.md

```diff
@@ -0,0 +1,191 @@
+---
+kep-number: 31
+title: Multicast Support
+authors:
+  - "@danwinship"
+owning-sig: sig-network
+reviewers:
+  - TBD
+approvers:
+  - "@thockin"
+editor: TBD
+creation-date: 2018-11-09
+last-updated: 2018-11-09
+status: provisional
+---
+
+# Multicast Support
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Existing Work](#existing-work)
+* [Proposal](#proposal)
+    * [User Stories](#user-stories)
+    * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+
+## Summary
+
+Currently, support for IP multicast traffic on the pod network is
+entirely undefined, with no multicast-related behavior being either
+required or forbidden, and different plugins offering different
+functionality. This KEP aims to standardize things.
+
+## Motivation
+
+Some Kubernetes users have workloads that require multicast traffic,
```

Comment:
I think it would be great to spell out some examples of multicast applications on Kubernetes and what their requirements are.